x-pack/plugins/infra/public/pages/metrics/inventory_view/components/layout.tsx

```diff
@@ -172,17 +181,24 @@ export const Layout = React.memo(
               </EuiFlexGroup>
             </TopActionContainer>
             <EuiFlexItem grow={false}>
-              <TryItButton
-                data-test-subj="inventory-hostsView-link"
-                label={i18n.translate('xpack.infra.layout.hostsLandingPageLink', {
-                  defaultMessage: 'Introducing a new Hosts analysis experience',
-                })}
-                link={{
-                  app: 'metrics',
-                  pathname: '/hosts',
-                }}
-                experimental
-              />
+              {!hostsLinkClickedRef?.current && (
```

Comment:
**nit**: we don't need the optional chaining on `hostsLinkClickedRef` as the ref obj is always defined, what can be missing is the `current` property being set to undefined if no initial value is passed to `useRef`

Reply:
That's correct, fixed.